pkg/cache/cache.go

```diff
@@ -94,39 +97,37 @@ func GetCacheFunc() (cache.NewCacheFunc, error) {
 	if infrastructure.IsOpenShift() {
 		// Annoying quirk: cache.SelectorsByObject uses an internal struct for values (internal.Selector)
 		// so we _can't_ just add Routes here since we cannot initialize the corresponding value.
```

Comment:
Could we delete this comment?
```
		// Annoying quirk: cache.SelectorsByObject uses an internal struct for values (internal.Selector)
		// so we _can't_ just add Routes here since we cannot initialize the corresponding value.
```

Reply:
This comment was added in this PR https://github.com/devfile/devworkspace-operator/pull/652

I'll remove it in next update.